plugins/search-react/src/extensions.tsx

```diff
@@ -105,9 +105,13 @@ const SearchResultListItemExtension = ({
   }, [rank, result, noTrack, analytics]);
 
   return (
-    <Box role="button" tabIndex={0} onClickCapture={handleClickCapture}>
+    <ListItem
```

Comment:
what do you think about this change @camilaibs?

Reply:
Looks great to me, I would only suggest adding a warning in the changeset set explaining if people have created a custom list item, they have to remove the <ListItem /> to prevent having a `<li/> inside of other <li/>...

Reply:
Thanks! Will rewrite the changesets based on the recent updates! 💯

Reply:
I believe <ListItem/> accepts a `button` prop. Does that change the role to button 🤔 ?

Reply:
Yeah good call out, I do not have 100% background context to why the button role was introduced here but I guess it made sense to use the `role="button"` prop on the `Box` component as thats just a normal div but had an action here, now that its a list element I believe we have control over the fact that it is a list item. But let me read up on recommendations here!